addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7387,7 +7396,7 @@ msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#16206"
-msgid "Failed to delete at least one file"
+msgid "Failed to delete at least one file.[CR]See log for more details."
```

Comment:
If we ever want to be come more userfriendly, things like "See log for more details." need to go. Also if you want consistency, we're using "Check log for more details" in other places. And if you're willing to change it, I'd vote to at least make this more userfriendly by using `More details about this issue can be found in the log.` or something like that. But which non-geek will ever look at the log, and geeks know to check the log.